### PR TITLE
[] Conda goodies

### PR DESCRIPTION
**Pull request by CamposContentful**

### CODE REVIEW

File: {{ cookiecutter.repo_name }}/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .PHONY: clean data lint requirements sync_data_to_s3 sync_data_from_s3 initialize_version_control
-
+.ONESHELL:
```
Comment (fugufisch) [inline {{ cookiecutter.repo_name }}/Makefile:2]: What's this?

File: {{ cookiecutter.repo_name }}/Makefile [hunk 2]
```diff
@@ -30,13 +28,15 @@ requirements:
 {% if cookiecutter.python_interpreter == 'poetry' %}
 	poetry install
 {% elif cookiecutter.python_interpreter == 'conda' %}
-	$(PYTHON_INTERPRETER) -m pip install -U pip setuptools wheel
-	$(PYTHON_INTERPRETER) -m pip install -r requirements.txt
+	conda install -n $(PROJECT_NAME) pip setuptools wheel conda-build mamba
+	mamba install -n $(PROJECT_NAME) -c conda-forge --file requirements.txt
+	source ~/miniconda3/etc/profile.d/conda.sh
+	conda activate $(PROJECT_NAME)
+	python -m pip install -e .
```
Comment (fugufisch) [inline {{ cookiecutter.repo_name }}/Makefile:35]: Compare this to `poetry install`. 😄

File: {{ cookiecutter.repo_name }}/pyproject.toml [hunk 3]
```diff
@@ -26,7 +26,3 @@ pre-commit = "^2.13"
 line-length = 88
 target-version = ['py39']
 include = '\.pyi?$'
-
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
```
Comment (fugufisch) [inline {{ cookiecutter.repo_name }}/pyproject.toml:32]: Sorry, but needs to stay for the poetry case. 🤷🏻‍♂️ 
Should be conditional on `cookiecutter.python_interpreter`

File: {{ cookiecutter.repo_name }}/Makefile (hunk 1 above)
Comment (CamposContentful) [reply {{ cookiecutter.repo_name }}/Makefile:2]: Run lines in the same  env

File: {{ cookiecutter.repo_name }}/Makefile (hunk 2 above)
Comment (CamposContentful) [reply {{ cookiecutter.repo_name }}/Makefile:35]: Hahaha, I made it extra granular  🤌

File: {{ cookiecutter.repo_name }}/pyproject.toml (hunk 3 above)
Comment (CamposContentful) [reply {{ cookiecutter.repo_name }}/pyproject.toml:32]: :shaking cat:

**Review by fugufisch — APPROVED** [review]

**Pull request merged**